ros_ws/launch/car.launch

```diff
@@ -2,6 +2,8 @@
 <launch>
     <arg name="joystick_type" default="ps3"/>
 
+    <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find racer_description)/urdf/racer.xacro' use_gpu:=true visualize_lidar:=true laser_sample_count:=1080"/>
```

Comment:
`use_gpu:=true visualize_lidar:=true laser_sample_count:=1080"`
Why are these needed? Shouldn't they be irrelevant outside of Gazebo?

Reply:
They are but xacro needs _some_ values for those to compile. Maybe it's possible to write some default values into the xacro file, idk.

Reply:
Fixed it. :P